core/templates/dev/head/components/AnswerGroupEditorDirective.js

```diff
@@ -48,11 +47,6 @@ oppia.directive('answerGroupEditor', [
           $scope.rulesMemento = null;
           $scope.activeRuleIndex = ResponsesService.getActiveRuleIndex();
           $scope.editAnswerGroupForm = {};
-          $scope.answerGroupIsDefault = ($scope.rules !== null);
-          
-          $scope.isCorrectnessFeedbackEnabled = function() {
-            return explorationCorrectnessFeedbackService.isEnabled();
```

Comment:
As we no longer need the explorationCorrectnessFeedbackService in this file, I think the service should be removed in the definition of the controller itself.

Reply:
Thanks, done!